### ISSUE TITLE
Whitelisting of 0x Addresses

### ISSUE THREAD

**Issue by m3rlin5ky** [soonaverse/soonaverse-dao#1042]
**Describe the bug**

Right now there is no possibility do whitelist specific 0x addresses for a NFT.

Within the crypto space it is quite common for a project/team to whitelist a 0x address so that the whitelisted persons are able to claim/buy-claim the NFT. Within the Soonaverse collection this is not possible at the moment. Even if we use badges, that allow users to claim a specific NFT the user can sell it and buy another one due to the collection allowance of a badge.

**Additional context**

I think the most creator/user friendly way/solution to implement whitelistings would be to add the 0x address in a separate column (wl.address) within the .csv file before the collection is being uploaded and add a whitelist filter function/or a whitelist page so people could easily see the NFT(s) they are whitelisted for.

**Potential value added**

Value added for Soonaverse creators
-> only the 0x address is needed to whitelist Soonaverse members
-> collection can't be gamed (e.g. multi-claim by badge)
-> standard procedure on other chains - so creators are familiar with the process of whitelisting a 0x address

Value added for Soonaverse members
-> overview over whitelistings (whitelisting page/filter function within the collection)
-> easy claim/claim-buy process of the whitelisted NFT(s)
-> Soonaverse/3rd parties can provide whitelisting/wl-airdrop notifications to wl-members

This issue was already discussed here: https://github.com/soonaverse/soonaverse-dao/discussions/989#discussion-4680256

**Comment by adamunchained** [comment]
I agree, I think we CSV this might not be a complicated implementation. We just need to figure out the visualization within the collection record.

@humaint - can you think about the design for this?

**Comment by humaint** [comment]
@adamunchained added div on dashboard "soonaverse/packages/ui/src/app/pages/dashboard/dashboard.page.html"
